File: src/test/java/com/hugo/PersonTest.java
package com.hugo;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {
    @Test
    public  void BMITest(){
        Person person  = new Person(79,1.83f);
        Assertions.assertEquals(23.589834f,person.BMI());
    }
}

File: src/test/kotlin/com/hugo/PersonTest.kt
package com.hugo.kotlin

import com.hugo.Person
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest{
    @Test
    fun BMITest(){
        val person = Person("Hugo",78f,1.83f);
        Assertions.assertEquals(78f/(1.83f*1.83f),person.BMI())
    }


}

